lib/rubygems/gemcutter_utilities.rb

```diff
@@ -274,7 +274,7 @@ def wait_for_otp(webauthn_url)
     thread.report_on_exception = false
 
     url_with_port = "#{webauthn_url}?port=#{port}"
-    say "You have enabled multi-factor authentication. Please visit #{url_with_port} to authenticate via security device."
+    say "You have enabled multi-factor authentication. Please visit #{url_with_port} to authenticate via security device. If you have OTP enabled and are unable to verify using webauthn, you can re-run the gem signin command with the `--otp [your_code]` option."
```

Comment:
Nit: I suggest swapping to say "if you can't verify using WebAuthn but have OTP enabled, ...".

Reply:
Fixed!